docs/UserGuide.md

```diff
@@ -283,8 +283,8 @@ Format: `filter KEYWORD [MORE_KEYWORDS]`
   e.g. `Apple Pear` will return `Eat apple`, `Buy pear`
 
 Examples:
-* `filter apple` returns `Buy apple` and `Make apple juice`
-* `filter orange pear` returns `Buy orange`, `Buy pear`<br>
+* `findt apple` returns `Buy apple` and `Make apple juice`
+* `findt orange pear` returns `Buy orange`, `Buy pear`<br>
```

Comment:
I think the Command Summary at the bottom should be updated to? For standardisation

Reply:
Gotcha